lib/kino/bridge.ex

```diff
@@ -250,6 +250,15 @@ defmodule Kino.Bridge do
     match?({:ok, _}, io_request(:livebook_get_evaluation_file))
   end
 
+  @doc """
+  Requests the child spec for proxy handler with the given function.
+  """
+  @spec get_proxy_handler_child_spec((Plug.Conn.t() -> Plug.Conn.t())) ::
+          Supervisor.child_spec() | {module(), term()} | module()
```

Comment:
We are the ones controlling both sides, so we can use a more precise type, looking at the Livebook side that's `{module(), term()}` :D